pkg/vf/virtio.go

```diff
@@ -202,7 +202,7 @@ func (dev *VirtioRng) AddToVirtualMachineConfig(vmConfig *VirtualMachineConfigur
 // https://developer.apple.com/documentation/virtualization/running_linux_in_a_virtual_machine?language=objc#:~:text=Configure%20the%20Serial%20Port%20Device%20for%20Standard%20In%20and%20Out
 func setRawMode(f *os.File) error {
 	// Get settings for terminal
-	attr, _ := unix.IoctlGetTermios(int(f.Fd()), unix.TIOCGETA)
+	attr, _ := unix.IoctlGetTermios(int(f.Fd()), unix.TIOCGETA) //#nosec G115
```

Comment:
It is the first case in the commit message: « it assumes casting an uintptr file descriptor to int will always be fine ». [`f.Fd()`](https://pkg.go.dev/os#File.Fd) is an `uintptr` file descriptor which we cast to `int`

Reply:
> I wonder why Go is using uintptr for Fd() when internally it is using int:

From the look of it, for Windows:

https://github.com/golang/go/blob/96d8ff00c2d6a88384863a656fb5e53716b614d3/src/os/file_windows.go#L34-L45
https://github.com/golang/go/blob/96d8ff00c2d6a88384863a656fb5e53716b614d3/src/internal/poll/fd_windows.go#L226-L227

where `Handle` is an `uintptr`: https://cs.opensource.google/go/x/sys/+/master:windows/syscall_windows.go;l=21

Reply:
Good points about adding comments explaining what G115 is and about the helper. It's only `func unixFd(fd uintptr) int` so that it can be used both in virtio.go and virtionet.go, even  if it's trivial, it makes the code easier to read.